package.json

```diff
@@ -31,9 +31,11 @@
     "@hookform/resolvers": "2.9.9",
     "@mantine/core": "5.5.5",
     "@mantine/hooks": "5.5.5",
-    "@tanstack/react-location": "3.7.4",
     "@mantine/notifications": "5.5.5",
+    "@tanstack/react-location": "3.7.4",
+    "ahooks": "3.7.2",
     "dotenv-cli": "6.0.0",
+    "js-cookie": "3.0.1",
```

Comment:
We don't need third-party in favor of [CookieStore API](https://developer.mozilla.org/en-US/docs/Web/API/CookieStore)

Reply:
but we don't have type support for it as described [here](https://github.com/microsoft/TypeScript/issues/47314)